Types/Entities/LevelBubble.js

```diff
@@ -80,7 +80,7 @@ function LevelBubble(spec) {
     drawOrder: LAYERS.levelBubbles,
   })
 
-  bubbletLevel.sendEvent('draw')
+  //bubbletLevel.sendEvent('draw')
```

Comment:
⚠️ we don't want to merge the PR with this line. It is just a hack to get the preview to load on iOS for now